pyproject.toml

```diff
@@ -22,8 +22,9 @@ classifiers = [
 ]
 dynamic = ["version"]
 dependencies = [
-    "fastapi-slim>=0.111.0",
-    "typing_extensions; python_version < '3.9'",
+    "fastapi>=0.100.0",
+    "pydantic>=2.0",
```

Comment:
⚠️ if we think veda will use this we should maybe relax this and make the code compatible with pydantic 1 (only the ConfigDict is from pydantic 2)